xbmc/guilib/guiinfo/SystemGUIInfo.cpp

```diff
@@ -631,6 +644,11 @@ bool CSystemGUIInfo::GetBool(bool& value, const CGUIListItem *gitem, int context
       }
       break;
     }
+    case SYSTEM_PVR_COUNT:
+    {
+      value = CServiceBroker::GetPVRManager().Clients()->EnabledClientAmount() > 1 ? true : false ;
```

Comment:
Why `> 1`?  Shoudn't this read `> 0`?

Please remove the blank after `false`.

Or you even shorten this line: `value = (CServiceBroker::GetPVRManager().Clients()->EnabledClientAmount() > 1);`

Reply:
> Why `> 1`? Shoudn't this read `> 0`?

In this section, a `bool` response is being prepared.  In the skin, we only want to show the PVR instance name if there are more than 1 PVRs registered.  If there is only 1 PVR, there is no confusion and no need to display the PVR instance name.
 
> Please remove the blank after `false`.

Too late.  Jenkins already beat you to that one.  I applied the DIFF file provided.

> Or you even shorten this line: `value = (CServiceBroker::GetPVRManager().Clients()->EnabledClientAmount() > 1);`

I will make that change.

Reply:
> In this section, a bool response is being prepared. In the skin, we only want to show the PVR instance name if there are more than 1 PVRs registered. If there is only 1 PVR, there is no confusion and no need to display the PVR instance name.

-1 for this. It is not logical the return `false` if there actually is an PVR instance available. 

Either we need to rename this bool to something like "PVR.HasMultipleInstances" or to remove this bool and do the math in the skin by obtaining the integer value and put a visible condition based on value being > 1.

I clearly prefer option 2 (skin does the math).

Reply:
The behaviour is clearly laid out in the documentation.
https://github.com/xbmc/xbmc/blob/77eacb8b9c665b7b5db0efa8892078213efec163/xbmc/GUIInfoManager.cpp#L1342

IMHO, having C++ return a bool reduces the load on the skin.  However, I will change this to a ‘the skin does the maths’ model tomorrow as you require.

Reply:
The skin had been updated to evaluate `Integer.IsGreater(System.PVRCount,1)` to decide whether or not to show the PVR instance name.

The Boolean return value for this label has been removed.  Following your ‘skin does the maths’ logic, if a skin needs to know if PVRs are enabled, it can evaluate `Integer.IsGreater(System.PVRCount,0)` or use `System.HasPVRAddon`.